### DISCUSSION TITLE
MultiChoice Enum constructor

### DISCUSSION THREAD

**Discussion by Avid29** [CommunityToolkit/Labs-Windows#276] [Proposals]
The ``ToolkitSampleMultiChoiceOptionAttribute`` could have a constructor that takes an ``Enum e`` in place of the choices, and generates the options using ``Enum.GetValues(e.GetType())``;

**Comment by Avid29** [comment]
If I understand the system correctly, it could then override ``TypeName`` which would make Enum conversion easier. Which seems important as Enums aren't currently supported it appears.

**Reply by michael-hawker** [reply]
Related to #149 and #207 as well.
